src/svgs/railmap/main/loop/loop-shmetro.tsx

```diff
@@ -83,9 +83,11 @@ const LoopSHMetro = (props: { bank_angle: boolean; canvas: CanvasType.RailMap |
 
     // all y_shares in branches will be 0
     const y_shares = { ...y_shares_loop, ...Object.fromEntries(loop_branches.flat().map(stn => [stn, 0])) };
+    // before: branch_spacing / 400 * svg_height (Chito)
+    const verticalPadding = ((branchSpacingPct * svg_height) / 300 / 400) * svg_height;
```

Comment:
This looks super weird, right? In fact, when I change it to `const verticalPadding = (branchSpacingPct * svg_height) / 400;`, everything seems almost the same. Probably because when `svg_height` is around 300-500, the common value for most templates, additional `/ 400 * svg_height` will be around 1. So I believe we could use a simpler calculation method.

Reply:
Removing the `/ 400 * svg_height` can indeed simplify the conversion - but this becomes a breaking change and we should check the display behaviour of the existing loop line templates.

Reply:
Yes, I tested the visual appearance after removal and it looks almost the same. Should be OK to remove in this pr.

Reply:
Fixed.